build/RELEASING.md

```diff
@@ -5,6 +5,16 @@ Only for Maintainers.
  1. Decide the tag name (version name) to be released. e.g. 1.4.0-beta.1,
     1.0.0-rc9.7 etc.
```

Comment:
Not directly related to this PR, but it will be great to remove this ugly example `1.0.0-rc9.7` and promote correct syntax with `.` just after the prerelease type.

Reply:
Updated